### PR TITLE
windows: move install script tool to calico directory and add install script to release artifacts

### PR DESCRIPTION
**Pull request by lmm**
## Description

Prior to the monorepo, in commit https://github.com/projectcalico/calico/commit/346127c49ed44d152bba2ac941cef2da30655339, we added a tool to generate the Calico for Windows install script and placed that in the node directory. This was needed at the time because the windows upgrade image needs the install script.

This PR moves the install script tool, template, and supporting make targets over to the calico directory. Make targets in node will copy over the script from the calico directory where the install script lives.

This PR also updates the release tool to upload the install script to the release.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by song-jiang — APPROVED** [review]
LGTM

File: hack/release/RELEASING.md [hunk 1]
```diff
@@ -205,6 +205,12 @@ When starting development on a new minor release, the first step is to create a
    git add calico/_includes/release-notes/<VERSION>-release-notes.md
    ```
 
+1. Update the Calico for Windows installation script.
+
+   ```
+   make -C calico install-calico-windows-script GIT_VERSION=<VERSION>
```
Comment (caseydavenport) [inline hack/release/RELEASING.md:211]: Would be nice to not need a manual step for this - can't it be handled by our templating logic in the same way our manifests work?

Comment (lmm) [reply hack/release/RELEASING.md:211]: Yeah you're right, I should just revert the install-calico-windows.ps1 script back to a Liquid template.

Comment (lmm) [reply hack/release/RELEASING.md]: I've fixed this now. The install script was reverted to https://github.com/projectcalico/calico/blob/29c434e2b5330a021da7bb998cb40d3a044bc22d/scripts/install-calico-windows.ps1 with some fixes over top like https://github.com/projectcalico/calico/blob/552dae20400055f6023d4378e1e416821322b333/calico/scripts/install-calico-windows.ps1

**Review by caseydavenport — APPROVED** [review]

**Pull request merged**